api/envoy/config/filter/http/admission_control/v2alpha/admission_control.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.admission_control.v2alpha;
+
+import "envoy/api/v2/core/base.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.admission_control.v2alpha";
+option java_outer_classname = "AdmissionControlProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.admission_control.v3alpha";
+
+// [#protodoc-title: Admission Control]
+// Admission Control :ref:`configuration overview
+// <config_http_filters_admission_control>`.
+// [#extension: envoy.filters.http.admission_control]
+
+message AdmissionControl {
+  // If set to false, the admission control filter will operate as a pass-through filter. If the
+  // message is unspecified, the filter will be enabled.
+  api.v2.core.RuntimeFeatureFlag enabled = 1;
+
+  // The time window over which the success rate is calculated. The window is rounded to the nearest
+  // second. Defaults to 120s.
```

Comment:
FWIW this seems super long. Also, naively I would think we would want a rolling window? Can you clarify how this works in terms of rolling vs. static windows, etc.?

Reply:
So this _is_ a sliding window. Each second, we're popping off the success rate information from 121s ago and replacing it with the SR data from 1s ago. If I understood the first question correctly, we both want the same thing w.r.t. the windows. There are no tumbling windows.

I just went with 120s because that's what was mentioned in the SRE handbook. I can run some experiments with different window sizes for the same traffic profile to get a better handle on what most people would want. I can put a TODO in code to make sure we're not stuck with an insane default.

Reply:
Yes that makes sense, I would just update the docs to make that a bit more clear.

Reply:
nit: I would still clarify that this is a sliding window at 1s granularity, etc.